apps/api/src/integration/plugins/discord.integration.ts

```diff
@@ -58,28 +58,34 @@ export class DiscordIntegration extends BaseIntegration {
   public async init(): Promise<void> {
   const integration = this.getIntegration<DiscordIntegrationMetadata>()
   
-  this.logger.log('Initializing Discord integration')
+  const { id: integrationRunId } = await this.registerIntegrationRun({
+    eventId: 'init', // Special identifier for initialization
```

Comment:
Event ID should come from the trigger event. Refer https://github.com/keyshade-xyz/keyshade/blob/9fba34814557a0fef4622c6ca3c820d3cd9d038f/apps/api/src/integration/plugins/vercel.integration.ts#L59-L62